### ISSUE TITLE
Cannot load Flickr dataset 

### ISSUE THREAD

**Issue by weihua916** [pyg-team/pytorch_geometric#4911]
### 🐛 Describe the bug

Flickr's numpy file seems to be corrupted?

```python
from torch_geometric.datasets import Flickr
dataset = Flickr(root = 'dataset/flickr')
```

```
Traceback (most recent call last):
  File "utils.py", line 206, in <module>
    train_dict, test_dict, num_classes = prepare_data('flickr')
  File "utils.py", line 99, in prepare_data
    dataset = Flickr(root = 'dataset/flickr')
  File "/dfs/user/weihuahu/anaconda3/lib/python3.8/site-packages/torch_geometric/datasets/flickr.py", line 36, in __init__
    super().__init__(root, transform, pre_transform)
  File "/dfs/user/weihuahu/anaconda3/lib/python3.8/site-packages/torch_geometric/data/in_memory_dataset.py", line 57, in __init__
    super().__init__(root, transform, pre_transform, pre_filter)
  File "/dfs/user/weihuahu/anaconda3/lib/python3.8/site-packages/torch_geometric/data/dataset.py", line 88, in __init__
    self._process()
  File "/dfs/user/weihuahu/anaconda3/lib/python3.8/site-packages/torch_geometric/data/dataset.py", line 171, in _process
    self.process()
  File "/dfs/user/weihuahu/anaconda3/lib/python3.8/site-packages/torch_geometric/datasets/flickr.py", line 70, in process
    x = np.load(osp.join(self.raw_dir, 'feats.npy'))
  File "/dfs/user/weihuahu/anaconda3/lib/python3.8/site-packages/numpy/lib/npyio.py", line 418, in load
    raise ValueError("Cannot load file containing pickled data "
ValueError: Cannot load file containing pickled data when allow_pickle=False
```

### Environment

* PyG version: 1.10.1
* PyTorch version: 2.0.3
* OS: Ubuntu 16.04
* Python version: 3.8.8
* CUDA/cuDNN version: cu102
* How you installed PyTorch and PyG: pip

**Comment by rusty1s** [comment]
I think this is fixed in 2.0.4 or master. Can you confirm?

**Comment by weihua916** [comment]
Works! Thank you!
